app/javascript/components/get-version.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+
+var lastValue = null;
+
+const getVersion = gql`
+  {
```

Comment:
Nit: I think specifying a name for the graphql query will provide a name in the apollo devs tools.

Reply:
+1 this is best practice

https://www.apollographql.com/docs/react/data/operation-best-practices/#name-all-operations

Reply:
can you use this same query as a query AND a subscription? I thought you needed to name subscriptions specifically.